Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,108 @@
+class Supervisor < Formula
+  desc "Process Control System"
+  homepage "http://supervisord.org/"
+  url "https://github.com/Supervisor/supervisor/archive/3.2.1.tar.gz"
+  sha256 "d9c0b17ec42ac6477e7ef0ad2e1f6f1597f855a4d9606c024ac40eb034d6e9ed"
+
+  resource "meld3" do
```

Comment:
Is it still necessary even if Supervisor will work on Python 2.5+? (and actually on Python 2.4 too if the [ElementTree](http://effbot.org/zone/element-index.htm) library is installed)